test-plan/eventing/control-plane/channel-lifecycle/services.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: conformance-events
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/knative.dev/eventing/cmd/event_display
```

Comment:
It would be nice if we can skip using Knative Serving, since that is not a pre-requisite afaik.